packages/dropdowns/src/elements/Autocomplete/Autocomplete.tsx

```diff
@@ -24,8 +24,6 @@ interface IAutocompleteProps extends HTMLAttributes<HTMLDivElement> {
   disabled?: boolean;
   /** Applies inset `box-shadow` styling on focus */
   focusInset?: boolean;
-  /** Indicates that the element's menu is open */
-  isOpen?: boolean;
```

Comment:
@hzhu I don't believe the `isOpen` props are doing anything across `Autocomplete`, `Multiselect`, or `Select`. Removing for added clarity – `Dropdown` is the only place where this prop can/should be controlled.